.github/workflows/dashboard-api-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: Dashboard Api Tests
+name: Dashboard Api
```

Comment:
Changed this because its redundant, i.e it will now be `Dashboard Api / Tests` instead of `Dashboard Api Tests / Test`